### PR TITLE
Add UserAgent suffix to PingOne client

### PR DESCRIPTION
**Pull request by patrickcping**
Adding a UserAgent suffix to the PingOne client, including the version of the CLI tool.

Also added lines to the gitignore to prevent unnecessary files in PR for local development

### CODE REVIEW

File: cmd/platform/export.go [hunk 1]
```diff
@@ -232,11 +233,18 @@ func initApiClient(ctx context.Context) (*sdk.Client, error) {
 		return nil, fmt.Errorf("unrecognized PingOne Region: %q. Must be one of: %q, %q, %q, %q", region, connector.ENUMREGION_AP, connector.ENUMREGION_CA, connector.ENUMREGION_EU, connector.ENUMREGION_NA)
 	}
 
+	userAgent := fmt.Sprintf("pingctl/%s", version)
+
+	if v := strings.TrimSpace(os.Getenv("PINGCTL_PINGONE_APPEND_USER_AGENT")); v != "" {
```
Comment (erikostien-pingidentity) [inline cmd/platform/export.go:238]: Is this environment variable meant to be configurable via all three methods? (config file, env var, or parameter?)

Comment (patrickcping) [reply cmd/platform/export.go:238]: The intention is that we don't include this env var in viper parameter logic as it's an edge case for integration in CICD pipelines.  The vast majority of users will not need to use this variable, but there is an internal use case that does

**Review by erikostien-pingidentity — APPROVED** [review]
Looks good 👍

File: cmd/platform/export.go (hunk 1 above)
Comment (erikostien-pingidentity) [reply cmd/platform/export.go:238]: Sounds good 👍

**Pull request merged**